cmd/promtool/main.go

```diff
@@ -326,3 +349,101 @@ func CheckMetrics() int {
 
 	return 0
 }
+
+// QueryInstant performs an instant query against prometheus server.
+func QueryInstant(url *url.URL, query string) int {
+	config := api.Config{
+		Address: url.String(),
+	}
+
+	// Create new client.
+	c, err := api.NewClient(config)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "error creating API client:", err)
+		return 1
+	}
+
+	// Run query against client.
+	api := v1.NewAPI(c)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+	val, err := api.Query(ctx, query, time.Now())
+	cancel()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "query error:", err)
+		return 1
+	}
+
+	fmt.Println(val.String())
+
+	return 0
+}
+
+// QueryRange performs a range query against a prometheus server.
```

Comment:
nit: "prometheus" -> "Prometheus" (hey, it's our brand! :P)

Reply:
addressed